src/components/navigation-bar/navigation-bar.tsx

```diff
@@ -1,18 +1,15 @@
 import * as React from 'react';
 import styled from "styled-components";
-import {RouterRoute, routes} from "../../navigation/root-router";
+import {RouterRoute, routes} from "src/navigation/root-router";
 import {useLocation} from "react-router-dom";
-import {color, screenSizes} from "../../theme";
-import {Button} from "../button/button";
+import {color} from "src/theme";
```

Comment:
to teraz nie działa z tym "src/costam/"

Module not found: Error: Can't resolve 'src/components/button/button' in '/Users/cezaryknapik/ITSharkz/bohun-react-mst-boilerplate/src/components/navigation-bar'